thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-metric-dimension/component.js

```diff
@@ -129,6 +158,14 @@ export default Component.extend({
       const updates = { [metricUrn]: true, [toBaselineUrn(metricUrn)]: true, [toCurrentUrn(metricUrn)]: true };
 
       onSelection(updates);
+    },
+
+    showExclusions() {
```

Comment:
Same here with the doc  comments. First glance i do not understand when and why you `forceShowExclusions`. What it means to force show?